sideboard/lib/_websockets.py

```diff
@@ -384,7 +384,9 @@ def make_caller(self, method):
         originating_ws = sideboard.lib.threadlocal.get('websocket')
         if client and originating_ws:
             sub = originating_ws.passthru_subscriptions.get(client)
-            if not sub:
+            if sub:
+                sub.method = method
+            else:
```

Comment:
This is a minor bugfix to the "passthru subscriptions" feature of Sideboard's websocket RPC functionality.  The bug was that previously we could update forwarded subscription parameters, but were unable to update the methods, so if you tried then only the parameters would change and not the method, which would result in an error!  (As with other bugfixes in this changeset, the unit tests have been updated to check for the bugfix.)